src/map/MapSidebar.js

```diff
@@ -97,6 +97,55 @@ function MapSidebarMain (props) {
   const handleShow = () => ref.current && onShow(ref.current.ref.current[
     `offset${isVertical ? 'Height' : 'Width'}`
   ]);
+  const [allReputableNumber, setallReputableNumber] = useState([]);
+  const [tentativeGrowth, setTentativeGrowth] = useState([]);
+  const CommunityCeremony = api.registry.getOrUnknown('CommunityCeremony');
+
+  // gets the current number of Reputables
+  useEffect(() => {
+    async function getnumRep () {
+      const [reputationLifetime, currentCeremonyIndex] = await Promise.all([
+        api.query.encointerCeremonies.reputationLifetime(),
+        api.query.encointerScheduler.currentCeremonyIndex()
+      ]);
+      const tempAllRepSet = new Set();
+      const promises = [];
+      for (let cIndex = currentCeremonyIndex - reputationLifetime; cIndex <= currentCeremonyIndex; cIndex++) {
+        const communityCeremony = new CommunityCeremony(api.registry, [cid, cIndex]);
+        promises.push(api.query.encointerCeremonies.participantReputation.keys(communityCeremony));
+      }
+      const arrayOfRreputables = await Promise.all(promises);
+      for (const reputables of arrayOfRreputables) {
+        for (const reputable of reputables) {
+          tempAllRepSet.add(reputable);
+        }
+      }
+      setallReputableNumber(tempAllRepSet.size);
+    }
+    getnumRep();
+  }, [allReputableNumber, setallReputableNumber, api, CommunityCeremony, cid]);
+
+  // gets the tentative growth of Reputatbles
+  useEffect(() => {
+    async function getTentativeGrowth (allReputableNumber) {
+      const currentCeremonyIndex = await api.query.encointerScheduler.currentCeremonyIndex();
+      const currentCommunityCeremony = new CommunityCeremony(api.registry, [cid, currentCeremonyIndex]);
+      const lastCommunityCeremony = new CommunityCeremony(api.registry, [cid, (currentCeremonyIndex - 1)]);
+      const [currentparticipantReputation, lastparticipantReputation] = await Promise.all([
+        api.query.encointerCeremonies.participantReputation.keys(currentCommunityCeremony),
+        api.query.encointerCeremonies.participantReputation.keys(lastCommunityCeremony)
+      ]);
+      const toberounded = ((currentparticipantReputation.length - lastparticipantReputation.length) / allReputableNumber);
+      return (allReputableNumber != null) ? (Math.round(toberounded * Math.pow(10, 2)) / Math.pow(10, 2)) : null;
```

Comment:
Sorry, I was uncertain on how we want this. The meaning of tentative growth should be: allReputables + allowedNewbies, which leads to:

`allReputatbles + floor(allReputables/api.consts.meetupNewbieLimitDivider)`

This however, excludes endorsements. Hence, I suggest labelling the field like: `Tentative Growth (excluding endorsements)`